spec/factories/targets.rb

```diff
@@ -6,5 +6,6 @@
     target_count 100
     unit Target.units[:submissions]
     conference
+    campaign
   end
```

Comment:
> Creating a target does not require you to have a campaign

Yes, it doesn't. However, I needed campaign to write tests for following lines:
```Ruby
    if unit == Target.units[:submissions]
      numerator = campaign.submissions_count
    elsif unit == Target.units[:registrations]
      numerator = campaign.registrations_count
    # more branch
    end
```

I suppose another way of achieving the same would be something like:
```Ruby
 registration_target = { create(:target, target_count: 10, unit: Target.units[:registrations]) }
 campaign = create(:campaign, conference:registration_target.conference )
 campaign.targets = registration_target
```
I think adding campaign to factory is cleaner. Let me know what do you prefer and I will change it.